cmd/proxy-server/server/server.go

```diff
@@ -208,6 +244,13 @@ func Run(ctx context.Context, opts ...Option) {
 		}
 	}
 
+	// If there's a debug port, we want to start it early
+	if rs.debughandler != nil {
```

Comment:
I'm not sure != nil allows the caller to disable this with the flag, without building their own main, but adding and != "" would allow them to turn it off with a flag flip.

Reply:
Sure, done.